server/api/v2/openapi.json

```diff
@@ -870,6 +870,17 @@
                           "description": "Whether the rule has been disabled/acked for all clusters or not.",
                           "type": "boolean",
                           "example": "false"
+                        },
+                        "resolution_risk": {
+                          "description": "Indicates the impact of the resolution steps on the cluster and other associated risks. Behaves in the same way as total_risk, 0 is returned when the rule doesn't have a resolution_risk defined.",
+                          "enum": [
+                            0,
```

Comment:
Just a nit-pick: what does 0 means? That the value is absent? Will it make more sense to use `null` in that case?